Makefile

```diff
@@ -100,10 +104,10 @@ publish: s3-release docker-release
 docker-publish: docker-build docker-push
 
 docker-build:
-	@if [ ! -f "./_tmp/clusterloader2" ]; then echo "downloading clusterloader2"; aws s3 cp --region us-west-2 s3://aws-k8s-tester-public/clusterloader2-linux-amd64 ./_tmp/clusterloader2; else echo "skipping downloading clusterloader2"; fi;
 	@if [ ! -d "${HOME}/go/src/k8s.io/perf-tests" ]; then echo "cloning perf-tests"; mkdir -p ${HOME}/go/src/k8s.io; pushd ${HOME}/go/src/k8s.io; git clone https://github.com/kubernetes/perf-tests.git; popd; else echo "skipping cloning perf-tests"; fi
+	@if [ ! -f "./_tmp/clusterloader2" ]; then echo "compiling clusterloader2"; mkdir _tmp; cd ${HOME}/go/src/k8s.io/perf-tests/clusterloader2; GOPROXY=direct GOOS=${AKT_OS} GOARCH=${AKT_ARCH} go build -o ./clusterloader2 ./cmd; cd -; cp ${HOME}/go/src/k8s.io/perf-tests/clusterloader2/clusterloader2 _tmp; else echo "skipping compiling clusterloader2"; fi;
 	@if [ ! -d "./_tmp/clusterloader2-testing-load" ]; then echo "copying clusterloader2/testing/load"; cp -rf ${HOME}/go/src/k8s.io/perf-tests/clusterloader2/testing/load ./_tmp/clusterloader2-testing-load; else echo "skipping copying clusterloader2/testing/load"; fi
```

Comment:
Can we move this into the container image? Add another stage to the build and then `COPY` the `clusterloader2` stuff?